build.gradle

```diff
@@ -94,10 +94,35 @@ allprojects {
             jvmTarget = jvmTargetVersion
         }
     }
+}
 
-    apply plugin: 'org.jlleitschuh.gradle.ktlint'
-    ktlint {
-        version = libs.versions.ktlint.get()
+spotless {
+    kotlin {
+        ktlint(libs.versions.ktlint.get())
+            .setEditorConfigPath("$projectDir/.editorconfig")
+        target("**/*.kt")
+        targetExclude("**/build/", "**/resources/", "plugins/openpgp-api-lib/")
+        endWithNewline()
```

Comment:
This rule had been covered by `insert_final_newline` in `.editorconfig`.

Reply:
Right, I'll create a follow up change